lint/capability_fields_analyser.go

```diff
@@ -74,6 +107,32 @@ var CapabilityFieldAnalyzer = (func() *analysis.Analyzer {
 					if !ok {
 						return
 					}
+
+					nt, ok := field.TypeAnnotation.Type.(*ast.NominalType)
+					if ok {
+						hasCapability, found := compositeIdsCapabilitiesPropery[nt.Identifier.Identifier]
+
+						if found {
```

Comment:
how about 
`if !ok { return }`
`if !found {return} 
//. the rest `

As far as I remember idiomatic go tends to avoid nested conditionals to enhance readability :)

Reply:
I agree with your comment, but I don't have this code on my machine. And I've just checked I don't see uncommitted changes. Let me push again my branch to GitHub and see.

Reply:
I agree with your comments but somehow I don't see this code on my machine. And when Ii try to push my branch nothing changes it says that it's up to date ... 


I'll try push force now.

Reply:
Yeah, seems like I forgot to push or needed to do a push force. Please accept my apologies, you would need to have another look as code is a bit different.